api/comments/serializers.py

```diff
@@ -111,6 +113,8 @@ def update(self, comment, validated_data):
                     comment.edit(content, auth=auth, save=True)
                 except PermissionsError:
                     raise PermissionDenied('Not authorized to edit this comment.')
+                except ValidationValueError:
+                    raise ValidationError('Ensure this field has no more than 500 characters.')
```

Comment:
We might not want to have such a specific error message since we aren't validating against the exact number of entered characters. Something more generic will do, like `"Comment too long"`. 

Also, there may be other validators that could raise a `ValidationValueError`, so it would be more future-proof to use the error message from the `ValidationValueError`. Something like

```python
except ValidationValueError as err:
    raise ValidationError(err.args[0])
```